src/commands/ScheduledElements.php

```diff
@@ -0,0 +1,174 @@
+<?php
+
+namespace lsst\nextbuilds\commands;
+
+use Craft;
+use craft\elements\Entry;
+use craft\helpers\Db;
+use lsst\nextbuilds\services\ElementStatusEvents;
+use lsst\nextbuilds\events\StatusChangeEvent;
+use yii\base\Event;
+use yii\base\Module;
+use yii\caching\CacheInterface;
+use yii\console\Controller;
+use yii\console\ExitCode;
+use yii\console\widgets\Table;
+use yii\helpers\BaseConsole as ConsoleHelper;
+
+class ScheduledElements extends Controller
```

Comment:
New Craft CLI command that will be called shortly after the top of each hour to pick up on new status changes. Triggered via a K8 CronJob